apps/11gist-blog/script.js

```diff
@@ -1,5 +1,30 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+  $.ajax('http://127.0.0.1:8080/apps/11gist-blog/api/gists.json', {
+    success: function(userPosts) {
+      userPosts.forEach(function(userGist) {
+        var userSubstr = userGist.description; 
+        var postString = "<li>" + 
+                         "<a href='#' data-url='" + userGist.url + "'>" + userGist.description.substring(6,userSubstr.length) + "</a>" + 
```

Comment:
Nicely done scrubbing your description. You can also pass in just the starting spot to the substring method and it will extend to length of the string. 

ex: `userGist.description.substring(6) + ..` here